lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -47,11 +47,56 @@ def tracer(self) -> opentelemetry.trace.Tracer:
 
 *Upgrading from `v0`:*
 
-If you are upgrading from `charm_tracing` v0, you need to do the following steps (assuming you already
+If you are upgrading from `charm_tracing` v0, you need to take the following steps (assuming you already
 have the newest version of the library in your charm):
-1) If you have a dependency on `opentelemetry-exporter-otlp-proto-grpc` in your project for charm tracing, replace
-it with `opentelemetry-exporter-otlp-proto-http>=1.21.0`.
-2) Update your `tracing_endpoint` property to point at `self.tracing.otlp_http_endpoint()`
+1) Add the following dependency to your charm project (or, if your project had a dependency
+on `opentelemetry-exporter-otlp-proto-grpc` only because of `charm_tracing` v0, you can replace it with):
+
+`opentelemetry-exporter-otlp-proto-http>=1.21.0`.
```

Comment:
Do we want to include this step even with PYDEPS defined? It is probably still needed for unit tests etc. Maybe we can explain this?

Reply:
Added this mention in this case.